lib/Page.js

```diff
@@ -72,6 +73,17 @@ function Page(pageConfig) {
  * Util Methods
  */
 
+function addContentWrapper(pageData) {
+  const $ = cheerio.load(pageData);
+  if ($.root().has(`#${CONTENT_WRAPPER_ID}`).length) {
+    return pageData;
+  }
+  $(`#${CONTENT_WRAPPER_ID}`).remove();
```

Comment:
Actually, this line doesn't really do anything right? Consider the situation:

* The document already has an element of id `#content-wrapper`: Line 79 returns it straight away, so this line doesn't get executed.
* The document does not have an element of id `#content-wrapper`: The flow of execution will come to here, but since no such element exist, this statement has no effect.

Reply:
My mistake.

`Line 78` is supposed to check if the root has a id of `content-wrapper`. It wasn't intended to check its contents if there exists an id of `content-wrapper`.